### ISSUE TITLE
Arguments that specify only a default value and no type fail.

### ISSUE THREAD

**Issue by michaelstaib** [ChilliCream/hotchocolate#84]
```csharp
d.Field(t => t.Test(default(string))).Argument("name", a => a.DefaultValue("foo")));
```

**Comment by michaelstaib** [comment]
Expected:
The argument descriptor should try to infer the type in the following order from:
1. the method parameter (if the method declares this argument explicitly)
2. the default value if it is native
3. the default value literal

**Comment by michaelstaib** [comment]
This bug is fixed as part of the #85 integration test bug fixes. We still need an explicit test for the ArgumentDescriptor.

**Comment by michaelstaib** [comment]
With the PR #98 comes the test that ensures this functionality.
